src/python/detectors/hardcoded_ip_address/hardcoded_ip_address.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=hardcoded-ip-address@v1.0 defects=1}
+def hardcoded_ip_address_noncompliant():
+    from socket import AF_INET, SOCK_STREAM, socket
+    sock = socket(AF_INET, SOCK_STREAM)
+    # Noncompliant: Unsafe Hardcoded ip address is used.
+    sock.bind(('193.168.14.31', 80))
+# {/fact}
+
+
+# {fact rule=hardcoded-ip-address@v1.0 defects=0}
+def hardcoded_ip_address_compliant():
+    ip_add = "1080:0::0::8:200C:131.107.129.8"
```

Comment:
What's this? I'm not sure about this detector, from the rule manifest, it says: `A hardcoded IP address can make your code vulnerable to denial of service attacks and IP address spoofing to bypass security checks`. But seems to be a hardcoded IP? And as far as I see this `ip_add` is not used anywhere? Why do you have it in the first place?

Reply:
Corrected the code example.

Reply:
Corrected the code example.